src/main/scala/org/globalforestwatch/summarystats/integrated_alerts/IntegratedAlertsSummary.scala

```diff
@@ -148,6 +170,29 @@ object IntegratedAlertsSummary {
             }
           })
 
+          var (intDistAlertDate, intDistConfidence): (Option[String], Option[String]) = 
```

Comment:
So theoretically this could change existing alert numbers significantly? E.g. alert numbers last year in an area might change if they're overridden with this? I would double check with Teresa this will be communicated to users during the release